pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -121,14 +121,14 @@ in stdenv.mkDerivation rec {
     # fix environment variable reading
     substituteInPlace SConstruct \
         --replace "env = Environment(" "env = Environment(ENV = os.environ,"
-   '' + lib.optionalString (versionAtLeast version "4.4" && versionOlder version "4.6") ''
```

Comment:
versionOlder is needed here anymore?

Reply:
This would only exclude mongoDB v4.5.. I'm not sure whether this was the intended purpose since 4.5 would have been a development release. These were explicitly tagged as not production ready by the official mongoDB resources: https://www.mongodb.com/docs/manual/reference/versioning/#historical-releases.
In addition, looking at `all-packages.nix` we don't seem to have been providing 4.5 historically anyway.